solr/solrj/src/test/org/apache/solr/client/solrj/impl/CloudHttp2SolrClientTest.java

```diff
@@ -93,6 +94,22 @@ public class CloudHttp2SolrClientTest extends SolrCloudTestCase {
   private static CloudSolrClient httpBasedCloudSolrClient = null;
   private static CloudSolrClient zkBasedCloudSolrClient = null;
 
+  private static LogListener entireClusterStateLogs;
+  private static LogListener collectionClusterStateLogs;
+  private static LogListener commandLogs;
```

Comment:
LogListener is Closeable; it's intended to be used with try-with-resources pattern.  By having a static field of them, you need to take care to close them.  I'm not sure if LogListener is used this way across our tests... not sure I like it.  

Any way by listening globally for the whole suite, isn't asserting it kind of difficult to reason about as different tests will do stuff?

Reply:
I can look to create a "separate" `httpBasedCloudSolrClient`, different than the one created in the Test class. This way the LogListener can be used within this test with the "try-with-reources" pattern.